docs/benchmark.md

```diff
@@ -12,86 +12,119 @@ It's important to look at the units than at the percentages. In general, RxDogTa
 
 Running the benchmark on a [Pixel 3](https://store.google.com/product/pixel_3_specs) yields the following results:
 
-### Event throughput (grouped by number of events)
+### Event throughput: grouped by number of events
 
 Measures the amount of time it takes for given number of elements to pass through the stream.
 
-#### 1 item (Observable)
-| Benchmark | Time (ns) | Time (ms) | Percent Increase |
-|----------|------------|-----------|------------------|
-|RxDogTagAndroidPerf.observable1_false | 212ns | 0.000ms|
-|RxDogTagAndroidPerf.observable1_true | 13,267ns | 0.013ms | 6158.02% |
-
-#### 1 item (Flowable)
-| Benchmark | Time (ns) | Time (ms) | Percent Increase |
-|----------|------------|-----------|------------------|
-| RxDogTagAndroidPerf.flowable1_false | 223ns | 0.000ms |
-| RxDogTagAndroidPerf.flowable1_true | 13,129ns | 0.013ms | 5787.44% |
-
-#### 1000 items (Observable)
-| Benchmark | Time (ns) | Time (ms) | Percent Increase |
-|---------- |-----------|-----------|------------------|
-| RxDogTagAndroidPerf.observable1000_false | 17,596ns | 0.018ms
-| RxDogTagAndroidPerf.observable1000_true | 156,953ns | 0.157ms | 791.98%
-
-#### 1000 items (Flowable)
-| Benchmark | Time (ns) | Time (ms) | Percent Increase |
-|---------- |-----------|-----------|------------------|
-| RxDogTagAndroidPerf.flowable1000_false | 17,854ns | 0.018ms
-| RxDogTagAndroidPerf.flowable1000_true | 143,646ns | 0.144ms | 704.56%
-
-#### 1_000_000 items (Observable)
-| Benchmark | Time (ns) | Time (ms) | Percent Increase |
-|---------- |-----------|-----------|------------------|
-| RxDogTagAndroidPerf.observable1000000_false | 17,078,596ns | 17.079ms
-| RxDogTagAndroidPerf.observable1000000_true | 161,099,912ns | 161.100ms | 843.29%
-
-#### 1_000_000 items (Flowable)
-| Benchmark | Time (ns) | Time (ms) | Percent Increase |
-|---------- |-----------|-----------|------------------|
-| RxDogTagAndroidPerf.flowable1000000_false | 17,790,262ns | 17.790ms
-| RxDogTagAndroidPerf.flowable1000000_true | 152,887,724ns | 152.888ms | 759.39%
-
-### Subscribe Cost (grouped by complexity)
+#### Simple: 1 item (Observable)
+| Benchmark | Time (ns) | Time (ms) |
+|----------|------------|-----------|
+| RxDogTagAndroidPerf.observable_simple[enabled=false,times=1,guardedDelegateEnabled=false]          |         532ns |   0.001ms |
```

Comment:
Minor nit: Can we shorten the name of the benchmark? Not sure how much it makes sense to have the `RxDogTagAndroidPerf.observable_simple` part anymore. Only seems to add noise.

Reply:
Just did that in 61cd7eb!

![image](https://user-images.githubusercontent.com/1361086/65479390-2f643f80-de5b-11e9-83bc-de49cba481d9.png)